ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -83,11 +83,12 @@ GRADLE_GLOBAL_ARGS: ((gradle-global-args))
 {%- endmacro %}
 
 {% macro all_gating_jobs() %}
-{%- for test in (tests) if not test.name=="StressNew" -%}
+- Benchmark
```

Comment:
This causes the 'Benchmark' job to be gated by the 'Benchmark' job. Which I believe is a failure on Concourse' part that they even let it occur instead of bailing out on deployment. Make an associated change to 'Benchmark' to not do this.